repo/packages/C/chronos/7/package.json

```diff
@@ -26,9 +26,9 @@
       "url": "https://github.com/mesos/chronos/blob/master/LICENSE"
     }
   ],
-  "preInstallNotes": "We recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for the Chronos Service.",
-  "postInstallNotes": "Chronos DCOS Service has been successfully installed!\n\n\tDocumentation: http://mesos.github.io/chronos\n\tIssues: https://github.com/mesos/chronos/issues",
-  "postUninstallNotes": "The Chronos DCOS Service has been uninstalled and will no longer run.\nPlease follow the instructions at https://docs.mesosphere.com/latest/deploying-services/uninstall/ to clean up any persisted state.",
+  "preInstallNotes": "The use of chronos has been DEPRECATED! If you still choose to use it, we recommend a minimum of one node with at least 1 CPU and 2GB of RAM available for the Chronos Service.",
+  "postInstallNotes": "[DEPRECATED] Chronos DCOS Service has been successfully installed!\n\n\tDocumentation: http://mesos.github.io/chronos\n\tIssues: https://github.com/mesos/chronos/issues",
+  "postUninstallNotes": "[DEPRECATED] The Chronos DCOS Service has been uninstalled and will no longer run.\nPlease follow the instructions at https://docs.mesosphere.com/latest/deploying-services/uninstall/ to clean up any persisted state.",
```

Comment:
Rather than just flagging this as `[DEPRECATED]` we should also directly point to the new Jobs & Dependencies feature in DC/OS 2.2.0 so users don't lose functionality.

Reply:
I think making that prominently visible is most important. 
Adding the note about DC/OS 2.2 makes sense, added it to the description.